conf.d/40-smtp.conf

```diff
@@ -1,2 +1,7 @@
 ; Prepared to link with https://hub.docker.com/r/tecnativa/postfix-relay
 smtp_server = $SMTP_SERVER
+smtp_port = $SMTP_PORT
+smtp_user = $SMTP_USER
+smtp_password = $SMTP_PASSWORD
+smtp_ssl = $SMTP_SSL
+email_from = $EMAIL_FROM
```

Comment:
🤔 Does this actually work? Isn't a record in `ir.mail.server` required?

cc   @pedrobaeza

Reply:
I will check in my test database .. but think I have no ir.mail.server

Reply:
I have to check this 4 params I just take odoo.conf from https://github.com/odoo/docker/blob/16d5988408434e4678f3bf6318aabad542dec4ee/11.0/odoo.conf.

server and port is working but never really try this 4 params.